blog/en/blog/2024/04/05/build-apisix-in-sles15.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to build APISIX in SLES 15"
+authors:
+  - name: "Oier Saizar"
+    title: "Author"
+    url: "https://github.com/osaizar"
+    image_url: "https://avatars.githubusercontent.com/u/9879984"
+keywords: 
+- Apache APISIX
+- SLES
+- Suse
+description: By reading this article, you will learn how to build Apache APISIX for Suse Linux Enterprise 15
+tags: [Ecosystem]
+---
+
+> By reading this article you will learn how to build Apache APISIX SLES 15 from source code.
+> The build process will be done in the [SLE BCI 15 SP5 Base Container](https://registry.suse.com/categories/bci/repositories/bci-bci-base-15sp5)
+
+<!--truncate-->
+
+## Install dependencies
+Before starting to build APISIX we need to install some dependencies needed to launch the build process:
+
+```shell
+zypper install -y git sudo make wget
+```
+
+## Clone the APISIX repository
+Next, we can clone the APISIX repository:
+
+```shell
+git clone https://github.com/apache/apisix.git
+cd apisix
+```
+
+## Modify the utils/install-dependencies.sh script
+Currently the `utils/install-dependencies.sh` script does not support SLES 15, so we will need to modify it slightly to add support for this distro:
+
+```shell
+wget https://raw.githubusercontent.com/osaizar/apisix/master/utils/install-dependencies.sh
```

Comment:
I think we are better off having the shell script in the blog post itself. WDYT @yzeng25?

Reply:
Sure, having the script in the blog would be good.